app/components/candidate_interface/complete_section_component.html.erb

```diff
@@ -1,8 +1,15 @@
-<%= form_with model: @application_form, url: @path, method: @request_method do |f| %>
-  <div class='govuk-form-group'>
-    <%= f.hidden_field @field_name, value: false %>
-    <%= f.govuk_check_box @field_name, true, multiple: false, label: { text: t(checkbox_label) } %>
-  </div>
+<%= form_with model: section_complete_form, url: path, method: request_method do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <%= render(review_component) %>
+
+  <%= f.govuk_radio_buttons_fieldset :completed, classes: 'govuk-!-width-two-thirds ', legend: { text: t('application_form.completed_question'), size: 'm' } do %>
+    <% if hint_text %>
+      <div class="govuk-hint"><%= hint_text %></div>
+    <% end %>
+    <%= f.govuk_radio_button :completed, true, label: { text: radio_button_label }, link_errors: true %>
+    <%= f.govuk_radio_button :completed, false, label: { text: t('application_form.incomplete_radio') } %>
+  <% end %>
```

Comment:
Should we use the [`govuk_collection_radio_buttons`](https://govuk-form-builder.netlify.app/form-elements/radios/) method from the formbuilder gem here?

Would save having to hardcode the HTML for the hint text.

Reply:
Some additional logic has been added to handle multiple hints so I don't think we use `govuk_collection_radio_buttons` in this instance ...

https://github.com/DFE-Digital/apply-for-teacher-training/pull/4576/commits/71603914c7278042dcfd53d0b0aae883659bd9bc#diff-50d730970ac17db67fd2b404df73aa94495376e376aff3e845abd36972a6cec5R8-R14

Reply:
Boo.. 😞  Could this be avoided by passing in the `hint` as pre-formatted HTML instead, I wonder?

Reply:
Yep cool - if it's not a blocker then I could look into this further after this PR is merged. I think passing in html into the form builder also feels a bit yucky to be honest (ideally the form builder component should just accept an array of hints and the component should deal with the markup?). Perhaps I can post an issue on the repo and provide a fix.